src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -450,10 +451,7 @@ AuraEffect::AuraEffect(Aura* base, uint8 effIndex, int32 const* baseAmount, Unit
     CalculateSpellMod();
 }
 
-AuraEffect::~AuraEffect()
-{
-    delete m_spellmod;
-}
+AuraEffect::~AuraEffect() { delete m_spellmod; }
```

Comment:
Code-styling

Should keep the same styling as before.

Reply:
Sorry, My editor has the prettier so that when I save it, I leave it in place.

Reply:
No worries